pandas/tools/util.py

```diff
@@ -173,6 +174,8 @@ def to_numeric(arg, errors='raise', downcast=None):
             values = arg.values
     elif isinstance(arg, (list, tuple)):
         values = np.array(arg, dtype='O')
+    elif isinstance(arg, Decimal):
```

Comment:
were you able to move addtl logic to is_scalar as well (IOW, did that break things irrespective of this change)?